lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -16,19 +16,20 @@ class TestSamplesNamespace(Commons):
     Unit testing for SamplesNamespace class.
     In a web service context, we access class functions mostly via the
     Flask test client rather than trying to directly invoke the class
-    constructor and `post` service.
+    constructor and `get` service.
     """
 
     @pytest.fixture(autouse=True)
     def _setup(self, client):
         super()._setup(
             cls_obj=SampleNamespace(client.config, client.logger),
-            pbench_endpoint="/datasets/namespace/iterations",
+            pbench_endpoint="/datasets/namespace/random_md5_string1/iterations",
             elastic_endpoint="/_search",
-            payload={"name": "random_md5_string1"},
             index_from_metadata="result-data-sample",
         )
 
+    api_method = API_METHOD.GET
```

Comment:
This should be removed, right? your `Commons._setup` will set `self.api_method` based on the schema, so also setting a class property like this is redundant and possibly confusing.